lib/screens/buy/buy.dart

```diff
@@ -0,0 +1,158 @@
+import 'dart:core';
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:fusecash/models/app_state.dart';
+import 'package:fusecash/models/views/cash_wallet.dart';
+import 'package:fusecash/screens/buy/business.dart';
+import 'package:fusecash/screens/send/send_amount.dart';
+import 'package:fusecash/widgets/bottombar.dart';
+import 'package:fusecash/widgets/main_scaffold.dart';
+
+class BuyPage extends StatefulWidget {
```

Comment:
Can you change `BuyPage` to `BuyScreen`?

Reply:
sure NP.